docs/framework/additional-apis/index.md

```diff
@@ -20,38 +20,38 @@ author: "mairaw"
 ms.author: "mairaw"
 manager: "wpickett"
 ---
-# Additional Class Libraries and APIs
+
+# Additional class libraries and APIs
+
 The .NET Framework is constantly evolving and in order to improve cross-platform development or to introduce new functionality early to our customers, we release new features out of band (OOB). This topic lists the OOB projects that we provide documentation for.  
   
- In addition, some libraries target specific platforms or implementations of the .NET Framework. For example, the
- [CodePagesEncodingProvider Class](CodePagesEncodingProvider.xml) class  makes code page encodings available to UWP apps developed using the .NET Framework. This topic lists these libraries as well.  
+In addition, some libraries target specific platforms or implementations of the .NET Framework. For example, the <xref:System.Text.CodePagesEncodingProvider> class makes code page encodings available to UWP apps developed using the .NET Framework. This topic lists these libraries as well.  
   
-## OOB projects  
+## OOB projects
   
-|Project|Description|  
-|-------------|-----------------|  
-|[System.Collections.Immutable](System.Collections.Immutable.xml)|Provides collections that are thread safe and guaranteed to never change their contents.|  
-|[System.Net.Http.WinHttpHandler](http://go.microsoft.com/fwlink/p/?LinkId=624833)|Provides a message handler for [HttpClient Class](HttpClient.xml) based on the WinHTTP interface of Windows.|  
-<!-- todo: fix link -->
-|`System.Numerics.Vectors1`|Provides a library of vector types that can take advantage of SIMD hardware-based acceleration.| 
-<!-- restore this when the doc is found 
-|[TPL Dataflow Library](../Topic/TPL%20Dataflow%20Library.md)|The TPL Dataflow Library provides dataflow components to help increase the robustness of concurrency-enabled applications.|  
--->
+| Project | Description |  
+| ------- | ----------- |  
+| <xref:System.Collections.Immutable> | Provides collections that are thread safe and guaranteed to never change their contents. |
+| [System.Net.Http.WinHttpHandler](http://go.microsoft.com/fwlink/p/?LinkId=624833) | Provides a message handler for <xref:System.Net.Http.HttpClient> based on the WinHTTP interface of Windows. |
```

Comment:
Rather than an fwlink, this should link to WinHttpHandler on docs. If <xref:System.Net.Http.WinHttpHandler> doesn't work, it's found in ../../xml/System.Net.Http/WinHttpHandler.xml.